[2024-04-02 13:22:57,519] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 13:22:57] "OPTIONS /bop?sonda=func HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 13:22:57,587] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 13:22:57] "DELETE /bop?sonda=func HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 13:23:00,707] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 13:23:00] "GET /valvulas HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 13:23:00,712] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 13:23:00] "GET /preventores HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 13:23:09,441] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 13:23:09] "POST /bop HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 13:23:12,991] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 13:23:12] "GET /bops HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:29:56,800] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.24.235.173:5000 - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:29:56,800] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:05,332] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:05] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:05,430] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:05] "GET /openapi/ HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:05,601] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:05] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:05,739] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:05] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:05,740] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:05] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:05,749] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:05] "[36mGET /openapi/static/images/apidoc.svg HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:07,391] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:07] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:07,397] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:07] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:07,639] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:07] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:07,700] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:07] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:08,147] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:08] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:30:08,191] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:30:08] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:31:08,297] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:31:08] "[33mGET /secao HTTP/1.1[0m" 404 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:31:16,623] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:31:16] "[32mGET /auth HTTP/1.1[0m" 308 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:31:16,630] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:31:16] "[31m[1mGET /auth/ HTTP/1.1[0m" 401 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:31:41,574] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:31:41] "[33mPOST /login HTTP/1.1[0m" 404 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:31:51,735] ERROR log_exception() L828  Exception on /auth/login [POST] - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py L828 
Traceback (most recent call last):
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
TypeError: login() missing 1 required positional argument: 'form'
[2024-04-02 15:31:51,737] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:31:51] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:35:02,460] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.24.235.173:5000 - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:35:02,461] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:35:06,306] ERROR log_exception() L828  Exception on /auth/login [POST] - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py L828 
Traceback (most recent call last):
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
TypeError: login() missing 1 required positional argument: 'form'
[2024-04-02 15:35:06,308] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:35:06] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:51:55,862] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.24.235.173:5000 - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:51:55,862] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
[2024-04-02 15:52:02,599] INFO _log() L96   127.0.0.1 - - [02/Apr/2024 15:52:02] "POST /auth/login HTTP/1.1" 200 - - call_trace=/home/slsn/repos/bop_land_api/.venv/lib/python3.10/site-packages/werkzeug/_internal.py L96  
